src/services/pulls/usePullsTeam.tsx

```diff
@@ -30,14 +30,14 @@ type PullStates = z.infer<typeof PullStatesSchema>
 
 const PullSchema = z
   .object({
-    pullId: z.number().nullable(),
+    pullId: z.number(),
     title: z.string().nullable(),
     state: PullStatesSchema,
     updatestamp: z.string().nullable(),
     author: z
       .object({
         username: z.string().nullable(),
-        avatarUrl: z.string().nullable(),
+        avatarUrl: z.string().url('not a valid url'),
```

Comment:
it will be easier for us not to track the error copy for each url field and rely on zod